MekHQ/src/mekhq/gui/dialog/OutstandingScenariosNagDialog.java

```diff
@@ -41,7 +41,7 @@ protected boolean checkNag(final Campaign campaign) {
         return !MekHQ.getMekHQOptions().getNagDialogIgnore(getKey())
                 && campaign.getActiveAtBContracts(true).stream()
                         .anyMatch(contract -> contract.getCurrentAtBScenarios().stream()
-                                .anyMatch(scenario -> (scenario.getDate() != null)
+                                .anyMatch(scenario -> (scenario != null) && (scenario.getDate() != null)
```

Comment:
I'm not seeing how this is possible, given it should error out in Mission::getCurrentAtBScenarios during the filter if we have a null scenario in the list.

Namely, we process it with `.filter(scenario -> scenario.getStatus().isCurrent() && (scenario instanceof AtBScenario))`, which would NPE out on any null inclusions...

Reply:
If you look at the logs in #2747, that's where the exception happens and that's the only way it's possible for a NPE to happen there.

Reply:
The updates to that function to make it non-null occurred ~3 months ago:
https://github.com/MegaMek/mekhq/blame/v0.49.2/MekHQ/src/mekhq/campaign/mission/Mission.java#L164-L168

That fix is in 49.2 per git tags, so unless they were using a snapshot, I'm with Windchild in that I'm not quite sure how we got here.

Reply:
Just opened #2748, and... the campaign file cannot be loaded because it's missing multiple campaign files. Further, there is no log included in that issue.

Reply:
I'm pretty sure the error is a duplicate of #2709 and thus fixed by #2711